src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -90,29 +92,21 @@ public void start(Future<Void> startFuture) throws Exception {
         OpenAPI3RouterFactory.create(vertx, "openapi.json", ar -> {
             if (ar.succeeded()) {
                 OpenAPI3RouterFactory routerFactory = ar.result();
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.SEND.toString(), this::send);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.SEND_TO_PARTITION.toString(), this::sendToPartition);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.CREATE_CONSUMER.toString(), this::createConsumer);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.DELETE_CONSUMER.toString(), this::deleteConsumer);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.SUBSCRIBE.toString(), this::subscribe);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.UNSUBSCRIBE.toString(), this::unsubscribe);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.ASSIGN.toString(), this::assign);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.POLL.toString(), this::poll);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.COMMIT.toString(), this::commit);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.SEEK.toString(), this::seek);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.SEEK_TO_BEGINNING.toString(), this::seekToBeginning);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.SEEK_TO_END.toString(), this::seekToEnd);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.HEALTHY.toString(), this::healthy);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.READY.toString(), this::ready);
-                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.OPENAPI.toString(), this::openapi);
-
-                routerFactory.addGlobalHandler(rc -> {
-                    int requestId = System.identityHashCode(rc.request());
-                    log.debug("[{}] Request from {}", requestId, rc.request().remoteAddress());
-                    rc.put("request-id", requestId);
-                    HttpUtils.logRequest(rc);
-                    rc.next();
-                });
+                routerFactory.addHandlerByOperationId(HttpOpenApiOperations.SEND.toString(), this.SEND);
```

Comment:
Each `HttpOpenApiOperation` knows its corresponding `HttpOpenApiOperations` constant (because it's passed to the constructor), so I would use that here to eliminate the possibility of mixing up the operation ids and each `HttpOpenApiOperation`. So something like:

```
routerFactory.addHandlerByOperationId(this.SEND.operationId(), this.SEND);
```